acl_loader/main.py

```diff
@@ -559,6 +559,19 @@ def convert_input_interface(self, table_name, rule_idx, rule):
 
         return rule_props
 
+    def validate_rule_fields(self, rule_props):
+        protocol = rule_props.get("IP_PROTOCOL", 6)
```

Comment:
>6 [](start = 49, length = 1)

I think the code should be okay, I just comment on readability.
At first glance, changing the default value for the same field is confusing. If the field is missing, you just skip all the check.
Suggest get the field value only once.